comfyui_aws_stack/comfyui_aws_stack.py

```diff
@@ -105,19 +136,6 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
             allow_all_outbound=True,
         )
 
-        # Allow inbound traffic on port 80
-        asg_security_group.add_ingress_rule(
-            ec2.Peer.any_ipv4(),
-            ec2.Port.tcp(80),
-            "Allow inbound traffic on port 80",
```

Comment:
if I remember correctly the port 80 is just a redirect to the port 443, so no real need to remove

Reply:
This SG is attached to EC2, and ECS task has its own SG attached to each Network Interface of task.
EC2 does not require inbound access since it is not access directly, but it needs outbound access to pull container image.